extensions/funqy/funqy-google-cloud-functions/deployment/src/main/java/io/quarkus/funqy/gcp/functions/deployment/bindings/FunqyCloudFunctionsBuildStep.java

```diff
@@ -47,4 +52,12 @@ public void init(List<FunctionBuildItem> functions,
     public void choose(FunqyConfig config, FunqyCloudFunctionsBindingRecorder recorder) {
         recorder.chooseInvoker(config);
     }
+
+    @BuildStep
+    public void markObjectMapperUnremovable(BuildProducer<UnremovableBeanBuildItem> unremovable) {
+        unremovable.produce(new UnremovableBeanBuildItem(
+                new UnremovableBeanBuildItem.BeanClassNameExclusion(ObjectMapper.class.getName())));
+        unremovable.produce(new UnremovableBeanBuildItem(
+                new UnremovableBeanBuildItem.BeanClassNameExclusion(ObjectMapperProducer.class.getName())));
```

Comment:
BTW, not mandatory, but I think that just making the `ObjectMapper` unremovable would be sufficient. The producer will follow.

Reply:
Just tested it and it didn't work if I remove the `ObjectMapperProducer` unremovable built item.